### PR TITLE
Chapter22 qa

### PR DESCRIPTION
**Pull request by codinglikeagirl42**

### CODE REVIEW

**Comment by codinglikeagirl42** [comment]
There is an issue going from 22.1 HTML Escaping to 22.2 What is Jinja3.  It skips to 22.3(which it has as 22.2) Chapter practice

**Comment by codinglikeagirl42** [comment]
Also on 22.6.6 Reusing Content Video Content, the linked video is Jinja2?  Not sure if we just want to add another note or redo?

File: src/chapters/more-flask/reusing-content.rst [hunk 1]
```diff
@@ -289,7 +289,7 @@ insert different HTML code. It is possible to build a base template with more
 of these spaces. However, that discussion is beyond the scope of this course.
 
 If you would like to research this topic on your own, a good place to start is
-the `Jinja2 documentation <https://jinja.palletsprojects.com/en/2.11.x/templates/#template-inheritance>`__.
+the `Jinja3 documentation <https://jinja.palletsprojects.com/en/2.11.x/templates/#template-inheritance>`__.
```
Comment (jimflores5) [inline src/chapters/more-flask/reusing-content.rst:292]: The link still points to the old version documentation.  Here's the new link:

https://jinja.palletsprojects.com/en/3.0.x/templates/#template-inheritance

File: src/chapters/more-flask/reusing-content.rst [hunk 2]
```diff
@@ -289,7 +289,7 @@ insert different HTML code. It is possible to build a base template with more
...
 
 Video Summary
```
Comment (jimflores5) [inline src/chapters/more-flask/reusing-content.rst:294]: I changed the video title in YouTube to just reference "Jinja" template extensions.  I did find one brief spot in the video that shows the link to the Jinja2 documentation.  However, it's not a big enough error to warrant redoing the whole clip or adding an admonition box.

**Review by jimflores5 — APPROVED** [review]
Thanks for fixing the outdated links and references.

I found one link that still needs an update, and I made a few suggestions for the admonition box on the Jinja3 intro page.  Other than that, we're good to go on this chapter!

File: src/chapters/more-flask/reusing-content.rst (hunk 2 above)
Comment (codinglikeagirl42) [reply src/chapters/more-flask/reusing-content.rst:294]: Cool, that works

File: src/chapters/more-flask/what-is-jinja3.rst [hunk 3]
```diff
@@ -1,19 +1,26 @@
...
-.. index:: ! Jinja2
+.. index:: ! Jinja3
 
-The Flask framework uses the `Jinja2 template language <https://codeburst.io/jinja-2-explained-in-5-minutes-88548486834e>`__.
-Jinja2 lets us include both variables and a little programming logic inside of
+The Flask framework uses the `Jinja3 template language <https://codeburst.io/jinja-2-explained-in-5-minutes-88548486834e>`__.
+
+.. admonition:: Note
+   
+   Jinja3 was released early 2021 and currently there is not much in the way of tutorials.  The above link is to Jinja2 but will still give you a good overview.
```
[the remark this answers is not in the document]
Comment (codinglikeagirl42) [reply src/chapters/more-flask/what-is-jinja3.rst]: Great notes, I moved it and changed the wording, what do you think?

**Pull request merged**